plugins/tracker-resources/src/utils.ts

```diff
@@ -685,6 +689,50 @@ export interface StatusStore {
   byId: IdMap<WithLookup<IssueStatus>>
   version: number
 }
+
+/**
+ * @public
+ */
+export async function moveIssueToSpace (
+  client: TxOperations,
+  doc: Doc,
+  space: Ref<Project>,
+  extra?: DocumentUpdate<any>
+): Promise<void> {
+  const hierarchy = client.getHierarchy()
+  const attributes = hierarchy.getAllAttributes(doc._class)
+  for (const [name, attribute] of attributes) {
+    if (hierarchy.isDerived(attribute.type._class, core.class.Collection)) {
+      const collection = attribute.type as Collection<AttachedDoc>
+      const allAttached = await client.findAll(collection.of, { attachedTo: doc._id })
+      for (const attached of allAttached) {
+        // Do not use extra for childs.
+        if (name === 'subIssues') {
```

Comment:
I would recommend to check for collection type is ref to class derived from Issue, so this code will not depend on text constant. Thanks.